modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -330,6 +330,40 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 			},
 			true,
 		},
+		{
+			"Msg fails its ValidateBasic: MsgTransfer has an empty receiver",
```

Comment:
tested locally by removing `ValidateBasic`, this doesn't actually fix the issue, because I believe it will fail to execute. We need a message which fails validate basic but succeeds in execution

Reply:
I wonder if we should first open an issue to convert this test to assert the error returned

Reply:
gotcha, let me look into it.

Reply:
ya looked into this a bit, we get the same error even if we convert to assert the error. `executeMsg` just goes through the normal Msg processing flow and will eventually also call `ValidateBasic` on the faulty message we create:

https://github.com/cosmos/cosmos-sdk/blob/095a64195aba917187bbb33570063bc675494bcd/baseapp/msg_service_router.go#L130-L134

Reply:
But wouldn't the returned error change if the validate basic check was removed?

Reply:
resolved in slack discussion: see parent issue edit for resolution.